step6/index.py

```diff
@@ -103,5 +160,4 @@ def full_name(self):
 del emp1.full_name
```

Comment:
@osama10 could you please write some description on `del` key word usage

Reply:
will do